core/src/main/java/org/apache/druid/data/input/InputStats.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.data.input;
+
+/**
+ * Tracks bytes read from an input source.
+ */
+public interface InputStats
+{
+  default void incrementProcessedBytes(long incrementByValue)
+  {
+
+  }
+
+  default long getProcessedBytes()
+  {
+    return 0;
+  }
```

Comment:
why are default methods needed here?

Reply:
RowIngestionMeters is marked as an extension point, which is where this is used. I can move the default impls there.

Reply:
I am not sure if anyone uses their own impl of RowIngestionMeters. I would have preferred not having these default impls altogether.

Reply:
Moved to `RowIngestionMeters` for now.